velox/functions/prestosql/aggregates/tests/ApproxDistinctTest.cpp

```diff
@@ -337,10 +359,11 @@ TEST_F(ApproxDistinctTest, memoryLeakInMerge) {
                 .finalAggregation()
                 .capturePlanNodeId(finalAgg)
                 .planNode();
-  auto expected = makeFlatVector(std::vector<int64_t>({50311}));
+  auto expected = makeFlatVector(std::vector<int64_t>({49810}));
   auto task = assertQuery(op, {makeRowVector({expected})});
+  // Should be significantly smaller than 500KB (the number before the fix).
```

Comment:
May be add a reasoning why it will be smaller than 500KB, instead of specifying before fix. Reader will read the new version and will get confused how we come up with this number.